ocpmodels/preprocessing/atoms_to_graphs.py

```diff
@@ -194,6 +215,14 @@ def convert(self, atoms: ase.Atoms, sid=None):
             data.fixed = fixed_idx
         if self.r_pbc:
             data.pbc = torch.tensor(atoms.pbc)
+        if self.r_data_keys is not None:
+            for data_key in self.r_data_keys:
+                if data_key in atoms.info:
```

Comment:
If someone specifies a property but it doesn't actually exist, we should raise an error rather than silently ignoring it...Otherwise people will be surprised when it comes to train and the property isn't there.

Reply:
That was my original implementation, but @janiceblue [requested otherwise](https://github.com/Open-Catalyst-Project/ocp/pull/622#discussion_r1504999495).

Reply:
went back to just letting it raise a KeyError